### PR TITLE
Fix #236 - allow + in the email

### LINKED ISSUE #236: allow '+' character in emails
The client-side email validation regex doesn't allow for the + symbol in email addresses, which is a valid character to use (and invaluable for testing!)

### PR DESCRIPTION
**Pull request by alicoding**
Fixes #236

### CODE REVIEW

**Comment by alicoding** [comment]
@cadecairos R?

File: templates/lib/regex/regex.test.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        should(!!"".match(regex.email)).be.equal(false);
+      });
+      it('Should fail for "ali@aa."', function () {
+        should(!!"".match(regex.email)).be.equal(false);
+      });
+      it('Should fail for "ali@aa.c"', function () {
+        should(!!"".match(regex.email)).be.equal(false);
+      });
+      it('Should fail for "ali@aa.googles"', function () {
+        should(!!"".match(regex.email)).be.equal(false);
```
Comment (cadecairos) [inline templates/lib/regex/regex.test.js]: fix up these tests

File: templates/lib/regex/regex.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  email: /^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/i,
+  password: {
+    bothCases: /^.*(?=.*[a-z])(?=.*[A-Z]).*$/,
+    digit: /\d/
+  }
+}
```
Comment (cadecairos) [inline templates/lib/regex/regex.js]: newline plz

**Comment by cadecairos** [comment]
R+

**Pull request merged**